### PR TITLE
Add support for closing partial position

### PR DESCRIPTION
**Pull request by camelpac**
Add support for this API: https://alpaca.markets/docs/api-documentation/api-v2/positions/#close-a-position.
According to the docs one could specify partial fractional qty or percentage. 

percentage seems to be currently unsupported, so it is commented out and will be commented in once api supports it.

For now, one could close partial fractional qty like this: 

```py
api.close_position("AAPL", 0.5)
```

Read the docs for more details.

### CODE REVIEW

**Review by gnvk — APPROVED** [review]
LGTM

**Comment by camelpac** [comment]
thanks. I will wait for the PR with websockets version to be merged, then merge this once since the pyup check doesn't pass with websokcets 8.1

**Pull request merged**
